Source/RageCommunity.Library.TestPlugin/Commands.cs

```diff
@@ -22,5 +25,36 @@ public static void Command_GetZoneNameForCurrentLocation()
         {
             Game.LogTrivial($"The name of the current zone is: {Game.LocalPlayer.Character.Position.GetZoneName()}");
         }
+
+        [ConsoleCommand("Rage Community Library spawn enum models")]
+        public static void Command_SpawnEnumModels()
```

Comment:
This command will spawn many characters, as far as I understand the code. 

One idea we could discuss: the model name as parameter with auto-completion, to avoid having 20 NPC around?

Reply:
The peds are immediately deleted after they are spawned.  I created the command to make sure the models and hashes in the enums could all be spawned without error.  I can remove the command since I don't know if anyone else would need to use it?

Reply:
Sure, or add a summary to make it clear that this is more a unit test than a real command, or remove it, whatever you prefer.